writeups/spacesos/README.md

```diff
@@ -0,0 +1,40 @@
+# Spacesos Writeup
+Есть сервис на дарте, который через `messages.dart` отправляет сообщения на сервис на php. В нём и будут все уязвимости.
```

Comment:
That is not true that all the vulns are in the PHP service.

Dart one is the one that should not allow to control the urlstring and causes SSRF.